test/framework/universal_app.go

```diff
@@ -220,11 +226,7 @@ func NewUniversalApp(t testing.TestingT, clusterName, dpName, mesh string, mode
 	app.allocatePublicPortsFor("22")
 
 	if mode == AppModeCP {
-		app.allocatePublicPortsFor("5678", "5680", "5681", "5682", "5685", "9901")
-	}
-
-	if dpName == AppEgress {
```

Comment:
So will Egress not have anything exposed? 🤔

Reply:
For dps we have this whole tunnel story: https://github.com/kumahq/kuma/blob/f85020df3d268bde3596cd8e5cfcdcc2cd4eefdb/test/framework/envoy_admin/tunnel/universal.go#L16 that runs the curls from inside the container.

So this was never actually needed (plus not very useful because we bind admin to localhost:9901 so the port-forward wouldn't work...)